src/main/java/com/example/ajaxonboardingproject/repository/MovieSessionRepository.kt

```diff
@@ -7,5 +7,5 @@ import java.time.LocalDate
 
 interface MovieSessionRepository : JpaRepository<MovieSession, Long> {
     @Query("from MovieSession m where m.movie.id = :movieId and date_format(m.showTime, '%Y-%m-%d') = :date")
-    fun findAvailableSessions(movieId : Long, date : LocalDate) : MutableList<MovieSession>
+    fun findAvailableSessions(movieId : Long, date : LocalDate) : List<MovieSession?>
```

Comment:
Why u have changed `MovieSession` to `MovieSession?` ?

Reply:
because there could not be available movie session in DB

Reply:
Are u sure it could return the list with nullable value instead of empty list?